dotCMS/src/test/java/com/dotmarketing/portlets/contentlet/model/ResourceLinkTest.java

```diff
@@ -111,17 +114,26 @@ private User mockLimitedUser(){
     public void test_Html_ResourceLink_Expect_Downloadable_No_Port_Number() throws Exception{
 
         final String mimeType = "text/html";
-        final String htmlFileName = "comments-list.html";
         final String path = "/application/comments/angular/";
         final String hostName = "demo.dotcms.com";
         final long languageId = 1L;
         final boolean isSecure = false;
 
+        final File file = FileUtil.createTemporalFile("comments-list", "html");
+        final String content = "This is a test temporal file";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "This is a test temporal file" appears 6 times in this file; the first occurrence is on line 123](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5372524)